js/utils/InitModules/InitOOJQuery.js

```diff
@@ -1,13 +1,14 @@
 require("./InitOO.js");
 
-if (!window.$)
-{
+if ($ !== undefined) {
+  $.noConflict();
+}
+if (!(jQuery && $)) {
+  window.jQuery = require('jquery');
   window.$ = require('jquery');
```

Comment:
I haven't really looked into where jquery is used inside our player, so this might not make sense, but have we confirmed whether we really need to set jquery at the window level? 

Ideally the player should only be using `OO.$` internally, so if this is the case then maybe all we need is something like:
```
if (!OO.$) {
  OO.$ = require('jquery');
  OO.$.noConflict(); // Not sure if this would be needed
}
```

In several cases I've seen that we wrap our code like this:
```
(function(OO,_, $) {
  // Our code goes here
}(OO, OO._, OO.$));
```
If we've done this in all of the instances in which we need JQuery, then `OO.$` would be shadowing the window `$` variable inside the closure, so we wouldn't have to worry about replacing it at the window level.

Reply:
@jpfranco 
Yes, it looks like only $ is used in the code. But if user adds jQuery from google (for example) before the player was called, jQuery function will be avaiable in the player (from users jQuery version). We can't be sure that jQuery wont be able in the player in future.

User wont be able to use his version of jQuery after ooyala player If uses this way to declare noConflict: OO.$.noConflict();

If you want to remove this window.$ = require('jquery');, after user declares jQuery you wont be able to use your version of jQuery.

Reply:
@KseniiaSokolovaEpam When we use just $ it's usually passed in at the beginning of the file. So we are always using OO.$.  Do we will need window.$ if we are never using it in the code? I don't see how a customer assigning window.$ is going to interfere with ours.

Reply:
@pilievOoyala  are you talking about something like this:
```
if (!OO.$) {
  OO.$ = require('jquery');
}
```
in this case a user will have problem with jQuery after ooyala player.

```
AFTER CALLING OOYALA 2
The jQuery variable is fine. jQuery version is 3.2.1 
The $ variable is BROKEN!
"$ === jQuery" is false
```